cves/CVE-2011-3192.yml

```diff
@@ -175,8 +187,10 @@ lessons:
     applies:
     note:
   distrust_input:
-    applies:
-    note:
+    applies: true
+    note: |
+        This vulnerability is almost entirely due to the user input being allowed 
+        and interpretted. If the range input was better validated from the beginning,
```

Comment:
"If the range input was better validated from the beginning," full sentence is cut off.